.travis.yml

```diff
@@ -11,4 +11,4 @@ install:
 - true
 
 script:
-- ginkgo -r
+- ginkgo integration_tests/
```

Comment:
I'm concerned that we won't know to update this when we add unit tests to this project. Instead we could do something like:

```sh
go test -v $(go list ./... | grep -v '/vendor/')
```

This also means that we don't need the ginkgo cli installed.